jmix-flowui/flowui/src/main/java/io/jmix/flowui/app/jmxconsole/MBeanInfoDetailView.java

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright 2022 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.flowui.app.jmxconsole;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import com.vaadin.flow.router.Route;
+import io.jmix.core.MessageTools;
+import io.jmix.core.Metadata;
+import io.jmix.flowui.DialogWindows;
+import io.jmix.flowui.UiComponents;
+import io.jmix.flowui.app.jmxconsole.model.ManagedBeanAttribute;
+import io.jmix.flowui.app.jmxconsole.model.ManagedBeanInfo;
+import io.jmix.flowui.component.details.JmixDetails;
+import io.jmix.flowui.component.grid.DataGrid;
+import io.jmix.flowui.component.textfield.TypedTextField;
+import io.jmix.flowui.kit.action.ActionPerformedEvent;
+import io.jmix.flowui.model.CollectionContainer;
+import io.jmix.flowui.model.InstanceContainer;
+import io.jmix.flowui.view.*;
+import io.jmix.flowui.view.navigation.UrlParamSerializer;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@Route(value = "system/mbeaninfo/:mbean", layout = DefaultMainViewParent.class)
+@ViewController("ui_ManagedBeanInfo.detail")
+@ViewDescriptor("mbean-detail-view.xml")
+@EditedEntityContainer("mbeanDc")
+public class MBeanInfoDetailView extends StandardDetailView<ManagedBeanInfo> {
```

Comment:
let's disable `PreventBrowserTabClosing`. All data is saved after change, so we won't lose anything.